src/main/resources/phrases_en_US.properties

```diff
@@ -75,6 +75,7 @@ QnVersion.intro=My current version is %s, \
 
 Reports.success=Done! FYI, the full log is [here](%s) (took me %[ms]s)
 Reports.failure=Oops, I failed. You can see the full log [here](%s) (spent %[ms]s)
+Reports.stop-fails=Sorry, I failed to stop the previous command, however it has next result:
```

Comment:
@xupyprmv what do you mean by `next result`? Maybe `the following result` would be better?